### PR TITLE
[Havoc Demon Hunter] Added Glaive Tempest to parser

### PR DESCRIPTION
**Pull request by philipsabri**
Glaive Tempest added to the parser and showing talent statistics.
image

### CODE REVIEW

File: src/parser/demonhunter/havoc/CHANGELOG.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { LeoZhekov, flurreN } from 'CONTRIBUTORS';
 import { change, date } from 'common/changelog';
 
 export default [
+  change(date(2020, 12, 24), 'Glaive Tempest added to the parser and are shown in Talent statistics', [flurreN]),
```
Comment (yajinni) [inline src/parser/demonhunter/havoc/CHANGELOG.js]: Keep it simple. Users read amit sometimes and get confused. Say like stat box added for glaive tempest talent. 

We already know what a process like that entails.

File: src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          return;
+        }
+        this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.GLAIVE_TEMPEST_DAMAGE), this.onDamageEvent);
+      }
+
+    onDamageEvent(event) {
+        this.damage += event.amount;
+      }
+
+    statistic() {
```
Comment (Sharrq) [inline src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js]: TalentStatisticBox is deprecated. You should be able to find examples of the new Statistic boxes in any of the mage talent modules.

Essentially you use Statistic instead and set the category to STATISTIC_CATEGORIES.TALENTS

File: src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        super(...args);
+        this.active = this.selectedCombatant.hasTalent(SPELLS.GLAIVE_TEMPEST_TALENT.id);
+        if (!this.active) {
+          return;
+        }
+        this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.GLAIVE_TEMPEST_DAMAGE), this.onDamageEvent);
+      }
+
+    onDamageEvent(event) {
+        this.damage += event.amount;
```
Comment (Sharrq) [inline src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js]: probably want to count damage that was absorbed as well

this.damage += event.amount + {event.absorb || 0);

**Review by Sharrq — CHANGES_REQUESTED** [review]

File: src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js (hunk 3 above)
Comment (philipsabri) [reply src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js]: Fixed

File: src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js (hunk 2 above)
Comment (philipsabri) [reply src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js]: Thanks for the feedback, updated accordingly.

**Review by yajinni — APPROVED** [review]

**Pull request merged**